hpx/runtime/actions/continuation.hpp

```diff
@@ -103,7 +104,7 @@ namespace hpx
 ///////////////////////////////////////////////////////////////////////////////
 namespace hpx { namespace actions
 {
-    class HPX_EXPORT continuation;
+    class HPX_API_EXPORT continuation;
```

Comment:
Whatever is preferred. The reason why I chose `HPX_API_EXPORT` here is because it was defined like that in hpx_fwd.hpp:
https://github.com/STEllAR-GROUP/hpx/pull/1711/files#diff-45f3f7b090970a53023ccaf8769f3a22L382